apps/website/src/content/docs/informationpanel.mdx

```diff
@@ -17,6 +14,33 @@ The information panel is a pattern used primarily in tables. It allows viewing o
 
 The info panel is reserved for complex tables, because it allows the user to gradually explore the content without overwhelming them with too many details. It also helps avoiding the unpleasant horizontal scrolling when the table gets too wide. Not every table requires an information panel.
 
+## Usage
+
+### Composition API
+
+The Information Panel component is made up of customizable subcomponents. Here’s what the basic usage looks like:
+
+```tsx
+<InformationPanelWrapper>
+  <Table ... /> // component to overlay
+
+  <InformationPanel isOpen={true}>
+    <InformationPanelHeader onClose={() => {}}>
+      <Text variant='subheading'>InfoPanel heading</Text>
+    </InformationPanelHeader>
+
+    <InformationPanelBody>
+      <Text>Metadata</Text>
+      <InformationPanelContent ... />
+      <Text>Attributes</Text>
+      <InformationPanelContent ... />
+    </InformationPanelBody>
+  </InformationPanel>
+</InformationPanelWrapper>
+```
+
+By default, the Information Panel component is closed. It can be opened by setting `isOpen={true}`. The orientation and size of the panel can be set using the `orientation` and `resizable` prop respectively.
```

Comment:
> The […] size of the panel can be set using the […] `resizable` prop.

This is wrong. `resizable` has nothing to do with setting the "size" of the panel.